### PR TITLE
address Joss review thurber II

### PR DESCRIPTION
**Pull request by matthiasdemuzere**
Addressing paper comments from #64, https://github.com/matthiasdemuzere/w2w/issues/64#issuecomment-1170107953:

> I think opening the paper with a statement about WUDAPT gives a false impression that the new contribution is WUDAPT. Working in a sentence or two about w2w prior to the current opening would alleviate this.

Thanks for this suggestion, a good one. I flipped the order of the two paragraphs in the summary. First on W2W, and only afterwards its relation to WUDAPT. This makes more sense indeed.

> "more simple" --> "simpler"

Adjusted

> "netcdf" --> "NetCDF" 

Adjusted

### CODE REVIEW

File: w2w/w2w.py [hunk 1]
```diff
@@ -489,6 +494,7 @@ def wrf_remove_urban(
                     south_north=i, west_east=j, land_cat=12
                 ).values
                 newluf[12, i, j] = 0.0
+        print(datetime.now() - startTime)
```
Comment (theendlessriver13) [inline w2w/w2w.py:497]: I think this is not supposed to be in here right?

**Review by theendlessriver13 — COMMENTED** [review]
I think we need to rebase this some of the commits are already merged?

File: w2w/w2w.py (hunk 1 above)
Comment (matthiasdemuzere) [reply w2w/w2w.py]: ah, that's clumsy. These were the test statements I used to check the efficiency of that routine. I removed them now.

File: w2w/w2w.py [hunk 2]
```diff
@@ -415,6 +415,7 @@ def wrf_remove_urban(
     print(len(dst_data.south_north))
     for i in dst_data.south_north:
         from datetime import datetime
```
Comment (theendlessriver13) [inline w2w/w2w.py]: oh this should also be removed

**Review by theendlessriver13 — COMMENTED** [review]
Ah I guess you missed one spot where you checked in the profiling you did.

Another thing: What about the binaries/netcdf files being removed?

Also, I rebased the branch on top of the current main branch, so the previously squashed commits don't accidentally show up in here and cause some conflicts.

File: w2w/w2w.py (hunk 2 above)
Comment (matthiasdemuzere) [reply w2w/w2w.py]: But this already removed now, no?

**Comment by matthiasdemuzere** [comment]
> Ah I guess you missed one spot where you checked in the profiling you did.

This is fine now, no?

> Another thing: What about the binaries/netcdf files being removed?

I am not sure what happened what those? That was not on purpose. Yet know I don't seem to manage to get them back?
 
> Also, I rebased the branch on top of the current main branch, so the previously squashed commits don't accidentally show up in here and cause some conflicts.

Ok, thanks!

**Review by theendlessriver13 — APPROVED** [review]
> I am not sure what happened what those? That was not on purpose. Yet know I don't seem to manage to get them back?

I think they were accidentally checked in in #60 so removing them is fine, since it's just the result from the example run.

**Pull request merged**